### PR TITLE
Fix: Include NFT collat while adjusting collat % in vault form

### PR DESCRIPTION
**Pull request by KMKoushik**
## Description
- LP NFT collat is not included in calculating the amount of ETH to remove / add as collat for given collat %(Adjust collat form).
- The same issue happens for Adjust debt form
- Fixed a bug where NFT can't be deposited.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## Checklist

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] Added video recordings if it is a UI change

https://user-images.githubusercontent.com/24666922/158928250-251af15f-17ef-4f80-86e9-0f16a4039fc4.mov

### CODE REVIEW

File: packages/frontend/pages/vault/[vid].tsx
Comment (alexisgauba) [inline packages/frontend/pages/vault/[vid].tsx:347]: deposited minus?

Comment (alexisgauba) [reply packages/frontend/pages/vault/[vid].tsx:347]: wdym don't include in total collat % ?

Comment (KMKoushik) [reply packages/frontend/pages/vault/[vid].tsx]: That's a bad comment sorry. Updated it now

**Review by alexisgauba — APPROVED** [review]

**Comment by KMKoushik** [comment]
Will merge after jotai PR merge

File: packages/frontend/pages/vault/[vid].tsx
Comment (yichiehliu) [inline packages/frontend/pages/vault/[vid].tsx:352]: Do we use BN to avoid potential calculation/conversion error?
```suggestion
    let lpCollatPercent = new BigNumber(0)
    // If NFT is deposited, Collateral amount from LP NFT should not be included. So target collat % - lp collat % will give actual collat to remove
    if (lpNftId) {
      const { collateral: uniCollat } = await getUniNFTCollatDetail(lpNftId)
      lpCollatPercent = uniCollat.div(debt).times(100)
    }
    const newCollat = new BigNumber(percent).minus(lpCollatPercent).times(debt).div(100)
```

**Review by yichiehliu — APPROVED** [review]

**Pull request merged**
